### ISSUE TITLE
Crash when inflate a layout with FragmentContainerView and use NavHostFragment

### ISSUE THREAD

**Issue by gouvinb** [phrase/phrase-android#65]
# Context

Here is my project configuration :

- Phrase version : 3.7.1
- Affected Google Library :
  - `androidx.fragment = "1.6.1"`
  - `androidx.fragment androidx-navigation = "2.7.4"`

With this configuration, the Activity crashes when the layout is inflated. While searching with the debugger, I noticed that a constructor of the `FragmentContainerView` was called when it should not be. Here is the code in question :

```kt
    /**
     * Do not call this constructor directly. Doing so will result in an
     * [UnsupportedOperationException].
     */
    @JvmOverloads
    public constructor(
        context: Context,
        attrs: AttributeSet?,
        defStyleAttr: Int = 0
    ) : super(context, attrs, defStyleAttr) {
        if (attrs != null) {
            var name = attrs.classAttribute
            var attribute = "class"
            context.withStyledAttributes(attrs, R.styleable.FragmentContainerView) {
                if (name == null) {
                    name = getString(R.styleable.FragmentContainerView_android_name)
                    attribute = "android:name"
                }
            }
            if (name != null && !isInEditMode) {
                throw UnsupportedOperationException(
                    "FragmentContainerView must be within a FragmentActivity to use $attribute" +
                        "=\"$name\""
                )
            }
        }
    }
```

<details>
<summary><h1>Log</h1> (in the spoiler)</summary>
  
```log
FATAL EXCEPTION: main
Process: app.sample.android, PID: 9795
java.lang.RuntimeException: Unable to start activity ComponentInfo{app.sample.android/app.sample.android.presentation.MainActivity}: android.view.InflateException: Binary XML file line #43 in app.sample.android:layout/activity_main: Binary XML file line #7 in app.sample.android:layout/content_main: Error inflating class androidx.fragment.app.FragmentContainerView
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3802)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3942)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2448)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8194)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: android.view.InflateException: Binary XML file line #43 in app.sample.android:layout/activity_main: Binary XML file line #7 in app.sample.android:layout/content_main: Error inflating class androidx.fragment.app.FragmentContainerView
Caused by: android.view.InflateException: Binary XML file line #7 in app.sample.android:layout/content_main: Error inflating class androidx.fragment.app.FragmentContainerView
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
	at android.view.LayoutInflater.createView(LayoutInflater.java:866)
	at androidx.appcompat.app.PhraseAppCompatDelegateKt.access$createViewCompat(SourceFile:2)
	at androidx.appcompat.app.PhraseAppCompatDelegate.createView(SourceFile:4)
	at androidx.appcompat.app.PhraseAppCompatDelegate$factory2$1.onCreateView(SourceFile:1)
	at android.view.LayoutInflater.tryCreateView(LayoutInflater.java:1073)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1009)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:973)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
	at android.view.LayoutInflater.parseInclude(LayoutInflater.java:1275)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1131)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:1138)
	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1096)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:694)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
	at app.sample.android.databinding.ActivityMainBinding.inflate(ActivityMainBinding.java:73)
	at app.sample.android.databinding.ActivityMainBinding.inflate(ActivityMainBinding.java:67)
	at app.sample.android.presentation.MainActivity.onCreate(MainActivity.kt:94)
	at android.app.Activity.performCreate(Activity.java:8621)
	at android.app.Activity.performCreate(Activity.java:8599)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3784)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3942)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
  at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2448)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:294)
	at android.app.ActivityThread.main(ActivityThread.java:8194)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
Caused by: java.lang.UnsupportedOperationException: FragmentContainerView must be within a FragmentActivity to use android:name="androidx.navigation.fragment.NavHostFragment"
	at androidx.fragment.app.FragmentContainerView.<init>(FragmentContainerView.kt:120)
	at androidx.fragment.app.FragmentContainerView.<init>(FragmentContainerView.kt:105)
	at androidx.fragment.app.FragmentContainerView.<init>(Unknown Source:11)
	... 37 more
```
</details>

**Comment by theSoenke** [comment]
Hi @gouvinb, thank you for providing this detailed report. This view inflation should now be resolved in version [3.7.3](https://github.com/phrase/phrase-android/releases/tag/3.7.3). Could you give it try with this release again?

**Comment by gouvinb** [comment]
Hi @theSoenke,

After a few tests on Android 11 and 14, I no longer have a crash.

Thank you for your responsiveness.
